### PR TITLE
Improve cmake format script

### PR DESCRIPTION
**Pull request by vyasr**
This PR ports some improvements from rapidsai/rmm#913.

### CODE REVIEW

File: cpp/scripts/run-cmake-format.sh [hunk 1]
```diff
@@ -16,19 +16,33 @@
...
+if [ -z ${CUDF_ROOT:+PLACEHOLDER} ]; then
+    CUDF_BUILD_DIR=$(git rev-parse --show-toplevel 2>&1)/cpp/build
+    status=$?
+else
+    CUDF_BUILD_DIR=${CUDF_ROOT}
+fi
+
+if ! [ ${status} -eq 0 ]; then
+    if [[ ${CUDF_BUILD_DIR} == *"not a git repository"* ]]; then
+        echo "This script must be run inside the rmm repository, or the CUDF_ROOT environment variable must be set."
```
Comment (ttnghia) [inline cpp/scripts/run-cmake-format.sh:36]: Comment needs to be updated.

Comment (vyasr) [reply cpp/scripts/run-cmake-format.sh]: Good catch, I missed one. Thanks.

**Review by ttnghia — APPROVED** [review]

**Review by bdice — APPROVED** [review]

**Review by robertmaynard — APPROVED** [review]

**Comment by vyasr** [comment]
rerun tests

**Comment by vyasr** [comment]
rerun tests

**Comment by vyasr** [comment]
@gpucibot merge

**Pull request merged**